src/analyzer/protocol/rdp/rdp-analyzer.pac

```diff
@@ -101,6 +101,20 @@ refine flow RDP_Flow += {
 		return true;
 		%}
 
+	function proc_rdp_client_security_data(csec: Client_Security_Data): bool
+		%{
+		if ( ! rdp_client_security_data )
+                        return false;
+	 
+			RecordVal* csd = new RecordVal(BifType::Record::RDP::ClientSecurityData);
+			csd->Assign(0, val_mgr->GetCount(${csec.encryption_methods}));
+			csd->Assign(1, val_mgr->GetCount(${csec.ext_encryption_methods}));
+    
+			BifEvent::generate_rdp_client_security_data(connection()->bro_analyzer(),
+								    connection()->bro_analyzer()->Conn(),
+								    csd);
```

Comment:
Do you mean to `return true;` at the end of this function? Might be nice to clean up the indentation a little as well because right now it suggests everything belongs to the if statement.